### DISCUSSION TITLE
[FEATURE REQ] Filters for recent activity log on dashboard

### DISCUSSION THREAD

**Discussion by jazha** [autobrr/autobrr#1682] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not really a problem, its more what I want shown on the main dashboard page at any time

**Describe the solution you'd like**
I would like an option in a filter for it to be shown in recent activity or not

**Describe alternatives you've considered**
An alternative would be to allow to filter recent activity and tell it what filters to show or not

**Additional context**
Nothing more

**Comment by luckylittle** [comment]
+1, I would like to see filter that matched the push in the dashboard. Like this:

![Screenshot from 2024-04-19 09-03-22](https://github.com/autobrr/autobrr/assets/8032159/63993654-dbce-4d61-9b63-b2cf50fa98ac)

Reason: With lots of filters, it is impossible to tell which filter triggered the push without going into logs. This would be useful for e.g. consolidating multiple filters into one etc.

**Comment by zze0s** [comment]
The filter can be seen on the action status popover, if you click the green/blue/gray box. Reason why it can't really be in the list itself is because multiple filters might run actions on the announce/release so there will be multiple that match.
